utils/frame/remote-externalities/Cargo.toml

```diff
@@ -13,8 +13,8 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-jsonrpsee-ws-client = { version = "=0.2.0-alpha.6", default-features = false }
-jsonrpsee-proc-macros = "=0.2.0-alpha.6"
+jsonrpsee-ws-client = { version = "0.2.0", default-features = false, features = ["tokio02"] }
```

Comment:
note, we have switched to tokio here and that's why the tests must be executed on tokio 0.2.

Reply:
fwiw they don't _need_ to. at some point that was the only compatible way. If they can work with asyc-std or a higher version, it should be trivial to change them.

Reply:
ok, I think it would be possible to use the `async-std tokio1 compat` for that but I suppose it should be fixed in `try runtime` or something right?

Reply:
Yes this is used in `try-runtime` as well.

Reply:
@niklasad1 / @kianenigma : Is adding this attribute and removing the tokio one what you mean by using `async-std` for the tests: https://docs.rs/async-std/1.6.0/async_std/attr.test.html

Reply:
I can update #8966 to do this, since it also uses 0.2.0

Reply:
hmm, I'm not sure exactly how it's spawned on the executor (tokio, async-std or whatever) in this case but yeah that should work if you use [async-std with the tokio1 feature flag](https://github.com/async-rs/async-std/blob/master/Cargo.toml#L61)

I would suggest to do in another PR because it might give you some headache ^^, you must be use `tokio1` on `ws-client` too.

Reply:
ok, I will hold off then